ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/KsqlRestApplication.java

```diff
@@ -163,7 +162,7 @@ public static SourceName getCommandsStreamName() {
       final StreamedQueryResource streamedQueryResource,
       final KsqlResource ksqlResource,
       final VersionCheckerAgent versionCheckerAgent,
-      final BiFunction<KsqlConfig, KsqlSecurityExtension, Binder> serviceContextBinderFactory,
+      final KsqlRestServiceContextBinderFunction serviceContextBinderFactory,
```

Comment:
rename to `serviceContextBinderFunction` ?

Reply:
Or maybe better to rename `KsqlRestServiceContextBinderFunction` to `KsqlRestServiceContextBinderFactory` as it is a factor